README.md

```diff
@@ -26,23 +26,18 @@ Your ideas are welcome [here](https://github.com/containers/youki/issues/10).
 
 [User Documentation](https://containers.github.io/youki/user/basic_setup.html#quick-install)
 
-# About the name
-
-youki is pronounced as /joʊki/ or yoh-key.
-youki is named after the Japanese word 'youki', which means 'a container'. In Japanese language, youki also means 'cheerful', 'merry', or 'hilarious'.
```

Comment:
I think we should keep this part, even though not strictly necessary.

Reply:
I agree :)

Reply:
okay!